var/spack/repos/builtin.mock/packages/patch/package.py

```diff
@@ -21,3 +21,9 @@ class Patch(Package):
     patch('bar.patch', when='@2:')
     patch('baz.patch')
     patch('biz.patch', when='@1.0.1:1.0.2')
+
+    patch(
+        'https://github.com/spack/spack/commit/cc76c0f5f9f8021cfb7423a226bd431c00d791ce.patch',
+        sha256='6057c3a8d50a23e93e5642be5a78df1e45d7de85446c2d7a63e3d0d88712b369',
+        when='@2:',
+    )
```

Comment:
This package was already used in a `test` which was asserting the list of shas. We need either to use another package or to update the failing tests (and check if there are others).

Reply:
Oops thanks. I’ll fix this.

Reply:
fixed